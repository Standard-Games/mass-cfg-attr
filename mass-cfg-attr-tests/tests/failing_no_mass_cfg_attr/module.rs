#![deny(unused_must_use, deprecated)]
#![allow(path_statements)] // disable warning for test

#[deprecated]
mod test {
    pub fn ok() {}
}

fn main() {
    // Violates unused_must_use and deprecated without mass_cfg_attr
   test::ok();
}
