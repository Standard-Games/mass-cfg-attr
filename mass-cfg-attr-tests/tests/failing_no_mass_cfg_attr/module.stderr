error: use of deprecated function `test::ok`
  --> tests/failing_no_mass_cfg_attr/module.rs:11:10
   |
11 |    test::ok();
   |          ^^
   |
note: the lint level is defined here
  --> tests/failing_no_mass_cfg_attr/module.rs:1:26
   |
1  | #![deny(unused_must_use, deprecated)]
   |                          ^^^^^^^^^^
